src/ui/component/rewardSummary/view.jsx

```diff
@@ -2,59 +2,45 @@
 import * as React from 'react';
 import Button from 'component/button';
 import CreditAmount from 'component/common/credit-amount';
-import BusyIndicator from 'component/common/busy-indicator';
 
 type Props = {
   unclaimedRewardAmount: number,
   fetching: boolean,
-  fetchRewards: () => void,
-  fetchRewardedContent: () => void,
 };
 
 class RewardSummary extends React.Component<Props> {
-  componentDidMount() {
-    this.props.fetchRewards();
-    this.props.fetchRewardedContent();
-  }
-
   render() {
     const { unclaimedRewardAmount, fetching } = this.props;
     const hasRewards = unclaimedRewardAmount > 0;
-
     return (
       <section className="card card--section">
         <header className="card__header">
-          <h2 className="card__title">
-            {__('Rewards')}
-            {fetching && <BusyIndicator />}
-          </h2>
-
-          <p className="card__subtitle">
-            {!fetching &&
-              (hasRewards ? (
-                <React.Fragment>
-                  {__('You have')}
-                  &nbsp;
-                  <CreditAmount inheritStyle amount={unclaimedRewardAmount} precision={8} />
-                  &nbsp;
-                  {__('in unclaimed rewards')}.
-                </React.Fragment>
-              ) : (
-                <React.Fragment>
-                  {__('There are no rewards available at this time, please check back later')}.
-                </React.Fragment>
-              ))}{' '}
-            <Button button="link" label={__('Learn more')} href="https://lbry.com/faq/rewards" />.
-          </p>
+          <h2 className="card__title">{__('Rewards')}</h2>
         </header>
 
+        <p className="card__subtitle">
+          {fetching && __('You have...')}
+          {!fetching && hasRewards ? (
+            <React.Fragment>
+              {__('You have')}
```

Comment:
While I know we've used this approach elsewhere, it's a bad practice and doesn't really work for i18n.

Is there any ability to inject nodes inside of an i18n string? e.g. `__('You have %amount% LBC', { amount: <span>5</span> })`

If not, let's use a comment tag so we can track where these exist. Something like `@i18nfixme`.

Reply:
Not with `y18n`. I would imagine this exists somewhere though